src/test/suite/extension.test.ts

```diff
@@ -8,4 +12,15 @@ suite('Extension Test Suite', () => {
 		assert.ok(vscode.extensions.getExtension(extensionId));
 		done();
 	});
+
+	test('by default, didact setting to open on startup should be false', async function (done) {
```

Comment:
I am struggling to understand why this test is using async / done. There is no async call inside as far as I can see.